packages/webservice/app/src/main/resources/models/tables/queryStats.hjson

```diff
@@ -0,0 +1,113 @@
+{
+    tables:
+    [
+        {
+            name: QueryStats
+            table: QueryStats
+            description:
+            '''
+            Carbon Query Stats Table
+            '''
+            cardinality: large
+            measures :
+            [
+                {
+                    name: durationInMillis
+                    type: DECIMAL
+                    definition: IF (durationInMillis IS NULL, -1, durationInMillis)
+                }
+                {
+                    name: rowsReturned
+                    type: DECIMAL
+                    definition: IF (rowsReturned IS NULL, -1, rowsReturned)
+                }
+                {
+                    name: bytesReturned
+                    type: DECIMAL
+                    definition: IF (bytesReturned IS NULL, -1, bytesReturned)
+                }
+            ]
+            dimensions :
+            [
+                {
+                    name: requestId
+                    type: TEXT
+                    definition: requestId
```

Comment:
Same definition comment for dimension definitions (wrap column names in {{ and }}).